docs/user-guide/cno-additional-interfaces.md

```diff
@@ -1224,6 +1226,23 @@ the NadVlanMap populated with the information from CSV file will be part of acc-
 ```bash
 acc-provision -a -f openshift-sdn-ovn-baremetal -u admin -p ”password” -c acc_provision_input.yaml -o acc-provision-output.yaml
 ```
+
+##### 8.2.5.2 Adding VLANs in NAD VLAN map case
+
+Modified excel sheet has to be converted to CSV format and path to the file has to be specified in the acc-provision-input.yaml as:
+
+```yaml
+chained_cni_config.vlans_file: "nad_vlan_map_input_v2.csv"
+```
+In this example, `nad_vlan_map_input_v2.csv` has been located in the current directory from which acc-provision will be executed.
+
+Also, metnion old CSV (or CSV used for last provisioning) file path in argumennt of acc-provision command as:
```

Comment:
nit: spell *mention* *argument*
Please mention: Without providing old nad-vlan-map file, vlan ranges in pool will be coalesced greedily, which may affect existing traffic, because existing pools would be deleted and recreated.
Please also add in the fabric vlan pools section: To add new vlans without affecting traffic, always add new vlans in a separate range in the fabricvlanpool CR, if directly editing the CR(If using nad-vlan-map, acc-provision addresses this: see section on NAD VLAN map case). If each vlan in the new range is expected to removed at some point of time, please add that vlan in a range by itself(eg: 600-600 or comma separated <>,600). Otherwise removal of any vlan in the range would affect traffic in the other vlans momentarily(since range cannot be modified and has to be deleted and recreated to remove a specific vlan in the range).